### PR TITLE
feat: move empty_objects_publisher

### PR DESCRIPTION
**Pull request by tkimura4**
Signed-off-by: tomoya.kimura <tomoya.kimura@tier4.jp>

## Description
move the description of empty_objects_publisher from dummy_perception_publisher.launch to simulator.launch

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [x] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

File: launch/tier4_simulator_launch/launch/simulator.launch.xml [hunk 1]
```diff
@@ -45,6 +45,15 @@
           <arg name="use_vector_map" value="true" />
         </include>
       </group>
+
+      <!-- publish empty objects instead of object recognition module -->
+      <group unless="$(var perception/enable_object_recognition)">
```
Comment (takayuki5168) [inline launch/tier4_simulator_launch/launch/simulator.launch.xml:50]: It seems this tag is inside `<group if="$(var perception/enable_object_recognition)">`, which will never be true.

Comment (tkimura4) [reply launch/tier4_simulator_launch/launch/simulator.launch.xml:51]: I fixed at [b0eabbb](https://github.com/autowarefoundation/autoware.universe/pull/613/commits/b0eabbb753cf59ffafc1fe70d7a154e81e23063f)

Comment (tkimura4) [reply launch/tier4_simulator_launch/launch/simulator.launch.xml:51]: I also fixed the indent.

**Comment by takayuki5168** [comment]
Looks good

**Review by takayuki5168 — APPROVED** [review]

**Pull request merged**